sigmastate/src/test/scala/sigmastate/utxo/DistributedSigSpecification.scala

```diff
@@ -309,23 +303,24 @@ class DistributedSigSpecification extends SigmaTestingCommons {
     val script = """atLeast(4, Coll(pubkeyA, pubkeyB, pubkeyC, pubkeyD, pubkeyE, pubkeyF, pubkeyG, pubkeyH))"""
     val prop = compile(env, script).asSigmaProp
 
-    //Alice, Bob, Gerard, and Hannah are signing, others are simulated
+    // Alice, Bob, Gerard, and Hannah are signing, others are simulated
```

Comment:
This is opportunity to improve diversity. Suggest to use at least one Asian, Black and pronoun name in the list.

Reply:
Not willing to do that.

Reply:
Do not understand you here. Not willing to do renaming (standard for all the cryptography textbooks and papers) also.